go/libraries/doltcore/diff/table_deltas.go

```diff
@@ -247,6 +251,24 @@ func (td TableDelta) HasHashChanged() (bool, error) {
 	return !toHash.Equal(fromHash), nil
 }
 
+// HasSchemaChanged returns true if the table schema has changed between the
```

Comment:
Is it more useful to define the add / drop case as true? That seems like a schema change to me

Reply:
Same question for above method

Reply:
good call; changed `HasSchemaChanged` and `HasHashChanged` so that they behave correctly even if you don't check for `IsAdd` and `IsDrop` first.